science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+
+Die angegebenen Zahlen stimmen gut mit den Erkenntnissen aus der [ereignisbezogenen Nutzerbefragung (EDUS)](../2021-08-02-science-blog-3) vom Frühjahr 2021 überein. Dort hatten ca. 13.000 von ca. 26.000 Teilnehmenden (die alle eine rote Warnung erhalten hatten) Angaben zu ihren Testergebnissen übermittelt. Von ihnen wurden insgesamt 5,9% positiv getestet. Der Anteil der Positiven lag bei den PCR-Tests bei 13%, bei den Antigen-Schnelltests bei 0,3% ([ebenda, Tabellen 15 und 17](../2021-08-02-science-blog-3/#42-testergebnisse)).
+
+*Die Warnungen durch die CWA erfolgen damit ebenfalls umfangreich, angemessen und zeitnah.*
+
+### 2.3 Zweck 3 – Risikoermittlung
+
+Etwa jede fünfte Person, die eine rote Warnung zum Zeitpunkt der Testregistrierung (PCR) erhalten hatte, wurde positiv auf SARS-CoV-2 getestet. Bei denjenigen ohne Risikomitteilung war der Anteil zum Teil wesentlich geringer (siehe Abbildung und [Blog 4, 6.5](../2021-10-15-science-blog-4/#65-zusammenhang-zwischen-risikobewertung-und-infektion)).
```

Comment:
"Risikoermittlung" makes no sense here. 

```suggestion
Etwa jede fünfte Person, die eine rote Warnung zum Zeitpunkt der Testregistrierung (PCR) erhalten hatte, wurde positiv auf SARS-CoV-2 getestet. Bei denjenigen ohne Risiko-Begegnung war der Anteil zum Teil wesentlich geringer (siehe Abbildung und [Blog 4, 6.5](../2021-10-15-science-blog-4/#65-zusammenhang-zwischen-risikobewertung-und-infektion)).
```

Reply:
Hi Tim, 

is it possible that you have misread the word? "Risikoermittlung" is not written there.
At this point, the term "Risikomitteilung" is used because it refers to the notification of a red warning triggered after a risk encounter.

Best regards
Tobias

Reply:
Hi Tobias,

yes I've misread the word! Sorry for any confusion caused!